models/foreignreference.go

```diff
@@ -0,0 +1,24 @@
+// Copyright 2022 Gitea. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package models
+
+import (
+	"code.gitea.io/gitea/models/db"
+)
+
+const ForeignTypeIssue = "issue"
```

Comment:
comment is necessary

Reply:
Done

---
[source](https://lab.fedeproxy.eu/forgefriends/forgefriends/-/merge_requests/36#note_1043)